userspace/libsinsp/sinsp.h

```diff
@@ -945,6 +945,33 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 
 	uint64_t get_lastevent_ts() const { return m_lastevent_ts; }
 
+	/*!
+	  \brief Simple set of syscalls that enforce minimum sinsp state set,
+	  plus some valuable syscalls.
+	  It has same effect of old `simple_consumer` mode.
+	*/
+	std::unordered_set<uint32_t> simple_ppm_sc_set();
```

Comment:
For all these methods I would use the same approach we have used for `enforce_sinsp_state_ppm_sc` so take a set as input and merge it with the syscalls we have to add. I imagine a client that has a unique set in his state and calls all these methods maybe one after the other

```c
io_ppm_sc_set(my_set);
net_ppm_sc_set(my_set);
```
If no set is provided a new empty one could be used as default like in `enforce_sinsp_state_ppm_sc`
WDYT? @FedeDP @leogr

Reply:
makes sense!